www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/Api/DiagnosticsController.php

```diff
@@ -52,7 +52,7 @@ public function configAction()
 
         // Since errors are handled by the caddy_diagnostics script and returned as json, check for an error key in the response
         if (isset($responseArray['error'])) {
-            return ["status" => "failed", "message" => $responseArray['message']];
+            return ["status" => "failed", "message" => _($responseArray['message'])];
```

Comment:
we do use `gettext()` for consistency

Reply:
also you can't translate something that's not a static string

Reply:
That makes sense, I thought I was clever. xD